<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>StereoMeasurementTool</class>
 <widget class="QWidget" name="StereoMeasurementTool">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1632</width>
    <height>861</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>(Stereo-) Bildmesstool</string>
  </property>
  <widget class="QGroupBox" name="gb_add_points">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>241</width>
     <height>71</height>
    </rect>
   </property>
   <property name="title">
    <string>Neue Punkte hinzufügen</string>
   </property>
   <widget class="QPushButton" name="pbn_add_point">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>40</y>
      <width>101</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Punkt hinzufügen</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="led_pt_number">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_pt_number">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>111</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Punktnummer</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="gp_point_manager">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>10</y>
     <width>371</width>
     <height>71</height>
    </rect>
   </property>
   <property name="title">
    <string>Punkte verwalten</string>
   </property>
   <widget class="QComboBox" name="cb_point_list">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>121</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pbn_save">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>40</y>
      <width>101</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Punkte exportieren</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_point_list">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>131</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Punktauswahl für Messung</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pbn_erase_point">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>40</y>
      <width>101</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Punkt löschen</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pbn_erase_all">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>14</y>
      <width>101</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Alles löschen</string>
    </property>
   </widget>
  </widget>
  <widget class="QGraphicsView" name="gv_left">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>120</y>
     <width>801</width>
     <height>721</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pbn_load_img_left">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>90</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Bild laden</string>
   </property>
  </widget>
  <widget class="QGraphicsView" name="gv_right">
   <property name="geometry">
    <rect>
     <x>820</x>
     <y>120</y>
     <width>801</width>
     <height>721</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pbn_load_img_right">
   <property name="geometry">
    <rect>
     <x>820</x>
     <y>90</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Bild laden</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
